service/src/main/java/bio/terra/tanagra/db/ReviewDao.java

```diff
@@ -47,7 +47,9 @@ public class ReviewDao {
               .displayName(rs.getString("display_name"))
               .description(rs.getString("description"))
               .size(rs.getInt("size"))
-              .created(rs.getTimestamp("created"));
+              .created(DbUtils.timestampToOffsetDateTime(rs.getTimestamp("created")))
+              .createdBy(rs.getString("created"))
```

Comment:
`.createdBy(rs.getString("createdBy"))`?

Reply:
Done, good catch. Review was the one artifact I didn't manually test, because I wasn't sure how to create a review in swagger.